### PR TITLE
Fixes #32063 - remove use-ntp parameter from timezone command

### PR DESCRIPTION
**Pull request by obol89**
Make "timezone" command one-liner without unnecessary if/else. Even if we use "use-ntp" half of existing code is doing the same thing.

More info and description https://projects.theforeman.org/issues/32063

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
ok to test

File: app/views/unattended/provisioning_templates/provision/kickstart_default.erb [hunk 1]
```diff
@@ -187,11 +187,8 @@ authselect --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downca
 <% else -%>
 authconfig --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downcase || 'sha256' %> --kickstart
 <% end -%>
-<% if use_ntp -%>
-timezone --utc <%= host_param('time-zone') || 'UTC' %>
-<% else -%>
+
```
Comment (lzap) [inline app/views/unattended/provisioning_templates/provision/kickstart_default.erb]: Please remove this empty line to make tests passing, we have snapshots which are compared to rendered content and it's currently failing:

```
Diff for /home/jenkins/workspace/test_develop_pr_core/database/postgresql/ruby/2.5/slave/fast/test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart default.snap.txt:
--- expected
+++ actual
@@ -18,6 +18,7 @@
 rootpw --iscrypted xybxa6JUkz63w
 firewall --service=ssh
 authconfig --useshadow --passalgo=sha256 --kickstart
+
 timezone --utc UTC 
```

Comment (obol89) [reply app/views/unattended/provisioning_templates/provision/kickstart_default.erb]: It has been removed with a new commit.

**Review by lzap — APPROVED** [review]
Looks okay to me, Foreman tests are passing and "Could not find gem 'maruku (~> 0.7)' in any of the gem sources listed in your Gemfile" is irrelevant. Squashing and merging, thanks!

**Review by ezr-ondrej — COMMENTED** [review]
Wasn't this about `ntp` not supporting the `--ntpservers` option? Won't this fail now with `host_param(use-ntp)=true`?

**Comment by obol89** [comment]
@ezr-ondrej , I've checked this once again. I've provisioned a new server with this parameter:
`timezone --utc America/Chicago --ntpservers our-own-ntp01.ourdomain.com`
NTP server has been provided with `host_param('ntp-server')`. After provisioning, I'm able to verify NTP works fine and I see my NTP server in the `/etc/ntp.conf` and in `ntpq -p` as well.
I was testing this with CentOS 7, so `use_ntp=true` because:
`use_ntp = host_param_true?('use-ntp') || (is_fedora && os_major < 16) || (rhel_compatible && os_major <= 7)`

**Comment by obol89** [comment]
I tried one more thing:
`timezone --utc UTC` without `host_param('time-zone')` and without `host_param('ntp-server')` but with `use_ntp=true` and it works good.

**Comment by ezr-ondrej** [comment]
Yeah, sounds we are safe then :) Thanks for checking @obol89! And for the nice cleanup :relaxed:

**Pull request merged**